src/app/modules/main/stickers/view.nim

```diff
@@ -0,0 +1,108 @@
+import NimQml, atomics, json, sets, strutils, tables, json_serialization
+
+import status/types/[sticker, pending_transaction_type]
+import ./models/[sticker_list, sticker_pack_list]
+import ./io_interface
+import ../../../../app_service/service/stickers/dto/stickers
```

Comment:
@jrainville view should not now about dto, at lease module.nim is the last place which may know about dto, since module is to one who maps a dto to a form we need for a view.